### PR TITLE
Mark indent_size 4 for XML

### PR DESCRIPTION
**Pull request by egorikftp**
Unify formatting: https://github.com/ComposeGears/Valkyrie/pull/137/commits/02717f8d0bcaa2209c52cd65db81bc2cf013089a

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -2,13 +2,12 @@ root = true
...
 [*]
 charset = utf-8
-indent_size = 2
+indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.{kt,kts}]
-indent_size = 4
```
Comment (Goooler) [inline .editorconfig:11]: I suggest we can unify all files to indent_size 2, to keep code clean and less nesting.

Comment (egorikftp) [reply .editorconfig:11]: As for me XML with indent size 2 looks strange. Google uses 4 as default

File: .editorconfig [hunk 2]
```diff
@@ -2,13 +2,12 @@ root = true
 
 [*]
 charset = utf-8
-indent_size = 2
+indent_size = 4
```
Comment (Goooler) [inline .editorconfig:5]: This also impacts yml or other files, we should leave them as indent_size for 2.

File: .editorconfig (hunk 1 above)
Comment (Goooler) [reply .editorconfig:11]: You can leave them as size 4 explicitly. I would prefer the global size 2.

Comment (egorikftp) [reply .editorconfig:11]: Ok, will leave default as 2 and make 4 for xml

Comment (Goooler) [reply .editorconfig:11]: #140

Comment (Goooler) [reply .editorconfig]: b9dbf75c7e437e70e74c0a2a4ab7713979aa5254

**Review by Goooler — APPROVED** [review]

**Comment by egorikftp** [comment]
Looks good 😊

**Pull request merged**